ports/nrf/boards/PCA10056/mpconfigboard.mk

```diff
@@ -2,6 +2,8 @@ MCU_SERIES = m4
 MCU_VARIANT = nrf52
 MCU_SUB_VARIANT = nrf52840
 SOFTDEV_VERSION = 6.1.1
+FLASHER ?= jlink
```

Comment:
Is this line needed, since jlink is the default flasher?

Reply:
Ah, not anymore that's true.